src/core/bst.h

```diff
@@ -787,126 +1030,130 @@ template <typename T> T* bst_cast(BST* node) {
 protected:
 public:
+    NoopBSTVisitor(bool skip_visit_child_cfg) : BSTVisitor(skip_visit_child_cfg) {}
     virtual ~NoopBSTVisitor() {}
 
-    virtual bool visit_arguments(BST_arguments* node) { return false; }
     virtual bool visit_assert(BST_Assert* node) { return false; }
-    virtual bool visit_assign(BST_Assign* node) { return false; }
     virtual bool visit_augbinop(BST_AugBinOp* node) { return false; }
-    virtual bool visit_attribute(BST_Attribute* node) { return false; }
     virtual bool visit_binop(BST_BinOp* node) { return false; }
-    virtual bool visit_call(BST_Call* node) { return false; }
-    virtual bool visit_clsattribute(BST_ClsAttribute* node) { return false; }
-    virtual bool visit_compare(BST_Compare* node) { return false; }
+    virtual bool visit_branch(BST_Branch* node) { return false; }
+    virtual bool visit_callattr(BST_CallAttr* node) { return false; }
+    virtual bool visit_callclsattr(BST_CallClsAttr* node) { return false; }
+    virtual bool visit_callfunc(BST_CallFunc* node) { return false; }
+    virtual bool visit_checkexcmatch(BST_CheckExcMatch* node) override { return false; }
     virtual bool visit_classdef(BST_ClassDef* node) { return false; }
-    virtual bool visit_delete(BST_Delete* node) { return false; }
+    virtual bool visit_compare(BST_Compare* node) { return false; }
+    virtual bool visit_copyvreg(BST_CopyVReg* node) { return false; }
+    virtual bool visit_deleteattr(BST_DeleteAttr* node) { return false; }
+    virtual bool visit_deletename(BST_DeleteName* node) { return false; }
+    virtual bool visit_deletesub(BST_DeleteSub* node) { return false; }
+    virtual bool visit_deletesubslice(BST_DeleteSubSlice* node) { return false; }
     virtual bool visit_dict(BST_Dict* node) { return false; }
     virtual bool visit_ellipsis(BST_Ellipsis* node) { return false; }
     virtual bool visit_exec(BST_Exec* node) { return false; }
-    virtual bool visit_expr(BST_Expr* node) { return false; }
-    virtual bool visit_extslice(BST_ExtSlice* node) { return false; }
     virtual bool visit_functiondef(BST_FunctionDef* node) { return false; }
-    virtual bool visit_index(BST_Index* node) { return false; }
+    virtual bool visit_getiter(BST_GetIter* node) override { return false; }
+    virtual bool visit_hasnext(BST_HasNext* node) override { return false; }
+    virtual bool visit_importfrom(BST_ImportFrom* node) override { return false; }
+    virtual bool visit_importname(BST_ImportName* node) override { return false; }
+    virtual bool visit_importstar(BST_ImportStar* node) override { return false; }
     virtual bool visit_invoke(BST_Invoke* node) { return false; }
-    virtual bool visit_keyword(BST_keyword* node) { return false; }
-    virtual bool visit_langprimitive(BST_LangPrimitive* node) { return false; }
+    virtual bool visit_jump(BST_Jump* node) { return false; }
+    virtual bool visit_landingpad(BST_Landingpad* node) override { return false; }
```

Comment:
Can you remove the `override` markers here or add them to all of the functions?  newer gccs complain.  same with FlattenVisitor and BasicBlockTypePropagator.

Maybe we should turn off `-Winconsistent-missing-override` until we can get that checked in CI :P